specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -1684,6 +1684,72 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoiceSections/{invoiceSectionName}/transactions": {
```

Comment:
>transactions [](start = 108, length = 12)

I noticed that this doesn't have a corresponding GET for a single transaction. I'm also noticing the same pattern on all the other ones that was already pre-approved, so I can't really block based on that.

However, this is a violation of the Resource Provider Contract. Can we get that fixed?